lab_1/main.py

```diff
@@ -69,7 +90,23 @@ def get_concordance(tokens: list, word: str, left_context_size: int, right_conte
     right_context_size = 3
     --> [['man', 'is', 'happy', 'the', 'dog', 'is'], ['dog', 'is', 'happy', 'but', 'the', 'cat']]
     """
-    pass
+    if not isinstance(tokens, list) or len(tokens) == 0 \
+            or not isinstance(word, str) or len(word) == 0 \
```

Comment:
No need for explicit check for length == 0.
You can just do:
`or not len(word)`